ant/org.eclipse.ant.tests.ui/Ant Tests/org/eclipse/ant/tests/ui/RunInSeparateThreadRule.java

```diff
@@ -0,0 +1,59 @@
+package org.eclipse.ant.tests.ui;
```

Comment:
Missing license header

Reply:
Added in 6ed46f883aec1f280fd75ec75ff72dd8a343cbf5.